var/spack/repos/builtin/packages/coin3d/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Coin3d(AutotoolsPackage):
```

Comment:
Why coin3d instead of coin?

Reply:
"coin" is a part (core library) of "coin3d". If you think that naming the package "coin" would be more correct, I will do it.